app/lib/actions/recurring_action.rb

```diff
@@ -10,11 +10,15 @@ def self.included(base)
     # Hook to be called when a repetition needs to be triggered. This either happens when the plan goes into planned state
     #   or when it fails.
     def trigger_repeat(execution_plan)
+      request_id = ::Logging.mdc['request']
+      ::Logging.mdc['request'] = nil
```

Comment:
After discussion with @ponrjek, we think the best way to support the recurring jobs would be to assign it a new (but non-empty) request id, so that we can group the related tasks from every iteration together (otherwise, we can do it just with the first iteration)

Reply:
Should I also log that?

Reply:
I canl handle the logging part in separate PR